substrate/frame/contracts/src/benchmarking/mod.rs

```diff
@@ -1162,6 +1163,300 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark(pov_mode = Ignored)]
+	fn set_transient_storage_empty() -> Result<(), BenchmarkError> {
+		let max_value_len = T::Schedule::get().limits.payload_len;
+		let max_key_len = T::MaxStorageKeyLen::get();
+		let key = Key::<T>::try_from_var(vec![0u8; max_key_len as usize])
+			.map_err(|_| "Key has wrong length")?;
+
+		let mut setup = CallSetup::<T>::default();
+		let (mut ext, _) = setup.ext();
+		let mut runtime = crate::wasm::Runtime::new(&mut ext, vec![]);
```

Comment:
nit can't we just use the macro here `build_runtime!(runtime, memory: [ ]);` same apply below I believe

Reply:
Yes, we can do that. There will be some additional, unnecessary calls, but the construction will be simpler.